### DISCUSSION TITLE
Can't setup connection

### DISCUSSION THREAD

**Discussion by ModemTester** [shadowsocks/shadowsocks-android#2839] [Q&A]
[exitcode78.txt](https://github.com/shadowsocks/shadowsocks-android/files/7653822/exitcode78.txt)
![Screenshot_2021-12-04-19-22-44-0971039486](https://user-images.githubusercontent.com/95528555/144707617-10c7a990-5c18-4081-b813-9e0cd7a7d1f6.png)

logcat attached.

BRs

Thanks

**Comment by dev4u** [comment]
`unsupported method, `auto` is not a supported method`
换个加密方式
